src/opcodes.c

```diff
@@ -20,6 +20,26 @@ int hexToDecimalMode(unsigned char hex) {
   return (hi * 10) + lo;
 }
 
+bool isAccumulatorAddressMode(unsigned int address) {
+  return address == 0;
+}
+
+void aslOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus)
+{
+  unsigned char current = isAccumulatorAddressMode(address)
+    ? cpu->accumulator
+    : readBus(address, bus);
```

Comment:
Do you know if we will need to use more of this code block?
If yes, what you think to extract this in a specific method?
Is it makes any sense?

Reply:
:+1: